### ISSUE TITLE
4.0.0 windows ingester will not start with active cache and available connecton

### ISSUE THREAD

**Issue by traetox** [gravwell/gravwell#59]
Appears to be some sort of race due to cache allowing muxer to start before it can get an IP.

The error we git is: `Failed to get Source IP from ingest muxer: No active connections`

Config snippet is:
```
[Global]
Ingest-Secret = fakesecret
Connection-Timeout = 0
Insecure-Skip-TLS-Verify = true
#note that backslashes (\) are an escape character and must be escaped themselves
Cleartext-Backend-Target=10.0.0.1:4023 #example of adding a cleartext connection
#Cleartext-Backend-Target=127.1.0.1:4023 #example of adding another cleartext connection
#Encrypted-Backend-Target=127.1.1.1:4024 #example of adding an encrypted connection
Ingest-Cache-Path="C:\\programdata\\gravwell\\eventlog\\events.cache"
Max-Ingest-Cache=1024 #Number of MB to store, localcache will only store 1GB before stopping.  This is a safety net
Log-Level=INFO
```

**Comment by djfritz** [comment]
https://github.com/gravwell/gravwell/blob/master/ingesters/winevents/processing.go#L353

This looks like it's simply an over eager sanity check at startup.

**Comment by djfritz** [comment]
This also affects networkLog, collectd, and kafka_consumer.

**Comment by djfritz** [comment]
Closed by #61
